.github/workflows/microsite-build-check.yml

```diff
@@ -43,3 +43,7 @@ jobs:
       - name: build microsite
         run: yarn build
         working-directory: microsite
+
+      - name: verify sidebars
+        run: yarn verify:sidebars
+        working-directory: microsite    
```

Comment:
Looks like a missing newline might be the problem, set your editor to add trailing newline automatically to avoid! :grin:

Reply:
wow